resources/packs/gcp/gcp.lr

```diff
@@ -17,17 +17,61 @@ alias gcp.bigquery = gcp.project.bigqueryService
 alias gcp.storage = gcp.project.storageService
 
 // GCP Cloud Organization
-gcp.organization @defaults("id") {
+gcp.organization @defaults("name") {
   // Organization ID
-  id string
+  id() string
   // Organization name
-  name string
+  name() string
   // Organization state
-  lifecycleState string
+  state() string
+  // Deprecated. Use `state` instead.
+  lifecycleState() string
   // Organization IAM policy
   iamPolicy() []gcp.resourcemanager.binding
   // Access approval settings
   accessApprovalSettings() gcp.accessApprovalSettings
+  // List of folders
+  folders() gcp.folders
+  // List of projects
+  projects() gcp.projects
+}
+
+// GCP Folders
+private gcp.folders {
+  []gcp.folder
```

Comment:
I discussed with @scottford-io and @arlimus, we want to return all recursive elements as default. Using `all` is not a good field name because it conflicts with the `.all` keyword.

```
private gcp.folders {
  []gcp.folder
  // list of direct folders
  children() []gcp.folder
```

Reply:
fixed